backend/testfiles/execution/stdlib/language-tools/semanticTokenization.dark

```diff
@@ -833,6 +833,29 @@ module TokenizeMatchExpression =
       (TokenType.Symbol, (2L, 11L), (2L, 13L))
       (TokenType.Keyword, (2L, 14L), (2L, 19L)) ]
 
+
+
+  ("match x with\n| 1L | 2L | 3L -> true \n| _ -> false" |> tokenize) =
+    [ (TokenType.Keyword, (0L, 0L), (0L, 5L))
+      (TokenType.VariableName, (0L, 6L), (0L, 7L))
+      (TokenType.Keyword, (0L, 8L), (0L, 12L))
+      (TokenType.Symbol, (1L, 0L), (1L, 1L))
+      (TokenType.Number, (1L, 2L), (1L, 3L))
+      (TokenType.Symbol, (1L, 3L), (1L, 4L))
+      //TODO missing symbol
```

Comment:
what symbol is missing here?

Reply:
Hmm, for  `| MPOr of Range * List<MatchPattern>`, I thought that `Range` would correspond to the whole thing, not the `|` symbol.

Here over in `MatchCase`,
```
      type MatchCase =
        { pat: Range * MatchPattern * Range
          whenCondition: Stdlib.Option.Option<Range * Expr>
          rhs: Expr }
```
, I thought that the 3rd part of `pat` would maybe capture the `|` part. Not sure how that fits into things.

In any case, two thoughts here:
- it's not that important to capture the `|` at this time, so probably ok to just add a CLEANUP and ignore this
- we should put a `///` comment on that `pat` to clarify what the tuple parts correspond to
- (bonus: that second bullet's point probably applies to a few more `Range`s, but I haven't looked to see)

Reply:
> Hmm, for | MPOr of Range * List<MatchPattern>, I thought that Range would correspond to the whole thing, not the | symbol.

That's correct, it does correspond to the whole thing (There are no pipe symbols captured in an `MPOr` currently)


And here  `pat: Range * MatchPattern * Range`
- the first range is for the `|`
- the second one is for the  `->`